viewing_party/party.py

```diff
@@ -1,23 +1,142 @@
 # ------------- WAVE 1 --------------------
 
+from tests.test_constants import GENRE_1, MOVIE_TITLE_1, RATING_1
+
+
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    if title and genre and rating:
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie
+    
+    return None
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+

```

Comment:
Awesome! These extra line breaks  aren't necessary for a function this small, but they won't cause any issues.